src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -26,6 +28,7 @@
 
 public final class StringHelper {
     private static final Logger log = Logger.getLogger();
+    private static final String HMAC_SHA1_ALGORITHM = "HmacSHA1";
```

Comment:
I'm not sure such a variable belongs in StringHelper. Since it's only used twice, let's follow the encrypt and decrypt methods and just use "HmacSHA1" instead.